### PR TITLE
Adding in a new job for platform engineer and modifying javascript engineer 

### PR DESCRIPTION
**Pull request by padraigobrien**
- Added in a new career page for platform engineer.
- Modified the javascript engineer page to reflect the pay in euro as opposed to dolars

### CODE REVIEW

File: src/pages/careers.js [hunk 1]
```diff
@@ -78,7 +78,14 @@ const OPEN_ROLES = [
       text: `Become a top contributor to the open-source software that Roadie is built on.`,
     },
   ],
-  [],
+  [
+    {
+      title() {
+        return <Link to="/careers/platform-engineer/">Platform Engineer</Link>;
+      },
+      text: `Come build Roadies platform`,
```
Comment (dtuite) [inline src/pages/careers.js:86]: I wonder can we talk about a benefit that the candidate will receive if they take the job here? This particular proposition doesn't jump out at me as extremely attractive.

Grammar wise, should it be "Roadie's platform"? I can never remember this rule.

File: src/pages/careers/platform-engineer.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  sitewideHeaderWrapper: {
+    marginBottom: 40,
+    paddingLeft: 16,
+    paddingRight: 16,
+  },
+}));
+
+const ROLE_NAME = 'Platform Engineer';
+const TYPEFORM_SLUG = 'platform-engineer';
+const HEADLINE = 'Help companies adopt backstage.io';
```
Comment (dtuite) [inline src/pages/careers/platform-engineer.js:22]: Personally, this headline makes me think of solutions engineer roles. I would point to the platform being built instead I think.

**Review by dtuite — CHANGES_REQUESTED** [review]

File: src/pages/careers.js (hunk 1 above)
Comment (padraigobrien) [reply src/pages/careers.js:86]: how about "take an active role in shaping the future of our platform"

Comment (dtuite) [reply src/pages/careers.js:86]: Sounds better to me.

Comment (padraigobrien) [reply src/pages/careers.js]: remedied to "Take an active role in shaping the future of our platform"

File: src/pages/careers/platform-engineer.js (hunk 2 above)
Comment (padraigobrien) [reply src/pages/careers/platform-engineer.js]: remedied to "Take an active role in shaping the future of our platform"

**Review by dtuite — APPROVED** [review]

**Pull request merged**